components/dashboard/Modal/Modal.tsx

```diff
@@ -0,0 +1,53 @@
+"use client";
+
+import Input from "@/components/common/Input/Input";
+import "./modal.scss";
+import { useState } from "react";
+import Button from "@/components/common/Button/Button";
+import { RxCross1 } from "react-icons/rx";
+import Link from "next/link";
+
+const Modal = () => {
+	const [hour, setHour] = useState<string>("");
+	const [name, setName] = useState<string>("");
+	const [dosage, setDosage] = useState<string>("");
+
+	const handleAdding = () => {};
```

Comment:
tu chyba coś zostało pominięte ?

Reply:
Tak, zostawiłam to sobie na później jak już zmerguję login i rejestrację, bo chcę to dodawać do bazy